packages/elements/src/header/index.ts

```diff
@@ -30,6 +31,8 @@ export class Header extends BasicElement {
     return VERSION;
   }
 
+  protected readonly defaultRole = 'heading';
```

Comment:
We should define defaultRole as `protected readonly defaultRole: string | null = 'heading';` so it can be override.

Reply:
We decided to developer handle attribute `role` and `aria-level` at the application level so the element header does not need to change.